SandB-iOS/SandB-iOS/Controllers/ArticlePageViewHolderController.h

```diff
@@ -1,5 +1,5 @@
 //
-//  NewArticlePageViewHolderController.h
+//  ArticlePageViewHolderController.h
```

Comment:
Let's remove these comment headers on all files. The info is included in our License and is redundant (and inaccurate)